### PR TITLE
Dashboard : display "No data" when no data available

### PR DESCRIPTION
**Pull request by Craaftx**
Link to this issue : https://github.com/Opla/front/issues/75

### CODE REVIEW

File: src/shared/containers/dashboard.jsx [hunk 1]
```diff
@@ -74,27 +82,43 @@ export class DashboardBase extends Component {
                 <Inner>
                   <Cell span={4}>
                     <div style={metricStyle}>
-                      <span style={valueStyle}>{metrics.users.count}</span>
+                      {Number.isNaN(Number(metrics.users.count)) ||
```
Comment (mikbry) [inline src/shared/containers/dashboard.jsx]: You should create a function that return this code. With metric value, unit as params. and reuse this function on all metrics.

File: src/shared/containers/dashboard.jsx [hunk 2]
```diff
@@ -30,7 +38,7 @@ const legendStyle = {
   textAlign: "center",
   fontSize: "16px",
   fontWeight: "400",
-  color: "#666",
+  color: "#666666",
```
Comment (mikbry) [inline src/shared/containers/dashboard.jsx]: I don't think it is a material color

File: src/shared/containers/dashboard.jsx [hunk 3]
```diff
@@ -21,7 +21,15 @@ const metricStyle = {
 const valueStyle = {
   fontSize: "34px",
   fontWeight: "500",
-  color: "#888",
+  color: "#888888",
```
Comment (mikbry) [inline src/shared/containers/dashboard.jsx]: I don't think it is a material color

**Comment by mikbry** [comment]
You need to launch tests before commiting, here the build failed due to dashboard test

File: src/shared/containers/dashboard.jsx (hunk 2 above)
Comment (willdurand) [reply src/shared/containers/dashboard.jsx]: mmh in any case `666` and `666666` are the exact same color.

Comment (Craaftx) [reply src/shared/containers/dashboard.jsx]: I write the colors as usual, I correct this

**Comment by Craaftx** [comment]
@mikbry @willdurand Ready for review

File: src/shared/containers/dashboard.jsx [hunk 4]
```diff
@@ -49,6 +57,21 @@ export class DashboardBase extends Component {
...
 
+  /* eslint-disable class-methods-use-this */
+  renderingValue(value, unit) {
+    let res;
+    if (Number.isNaN(Number(value)) || value === null) {
+      res = <span style={errorStyle}>No data</span>;
+    } else {
+      res = (
+        <span style={valueStyle}>
+          {value} {unit}
```
Comment (willdurand) [inline src/shared/containers/dashboard.jsx]: I think value and unit should not be separated with a space

**Comment by mikbry** [comment]
Dorian where are you on this PR ?

**Comment by Craaftx** [comment]
I forget to commit this morning..

**Comment by mikbry** [comment]
What is the status of this PR ?

**Pull request merged**